pkg/crc/constants/constants.go

```diff
@@ -71,18 +71,20 @@ func GetAdminHelperURL() string {
 	return adminHelperURLForOs[runtime.GOOS]
 }
 
-var defaultBundleForOs = map[string]string{
-	"darwin":  fmt.Sprintf("crc_hyperkit_%s.crcbundle", version.GetBundleVersion()),
-	"linux":   fmt.Sprintf("crc_libvirt_%s.crcbundle", version.GetBundleVersion()),
-	"windows": fmt.Sprintf("crc_hyperv_%s.crcbundle", version.GetBundleVersion()),
+func defaultBundleForOs(bundleVersion string) map[string]string {
+	return map[string]string{
+		"darwin":  fmt.Sprintf("crc_hyperkit_%s.crcbundle", bundleVersion),
+		"linux":   fmt.Sprintf("crc_libvirt_%s.crcbundle", bundleVersion),
+		"windows": fmt.Sprintf("crc_hyperv_%s.crcbundle", bundleVersion),
+	}
 }
 
-func GetDefaultBundleForOs(os string) string {
-	return defaultBundleForOs[os]
+func GetDefaultBundleForOs(os, bundleVersion string) string {
+	return defaultBundleForOs(bundleVersion)[os]
 }
 
```

Comment:
Could be 
```
var defaultBundleForOs = map[string]string{
	"darwin":  GetBundleName("darwin", version.GetBundleVersion()),
	"linux":   GetBundleName("linux", version.GetBundleVersion()),
	"windows": GetBundleName("windows", version.GetBundleVersion()),
}

func GetDefaultBundleForOs(os string) string {
	return defaultBundleForOs[os]
}

func GetBundleName(os, bundleVersion string) string {
    switch os {
        case "darwin": return fmt.Sprintf("crc_hyperkit_%s.crcbundle", bundleVersion)
        ....
    }
}
```

Reply:
@cfergeau how about then doing something like?

```
func bundleForOs(bundleVersion string) map[string]string {
	return map[string]string{
		"darwin":  fmt.Sprintf("crc_hyperkit_%s.crcbundle", bundleVersion),
		"linux":   fmt.Sprintf("crc_libvirt_%s.crcbundle", bundleVersion),
		"windows": fmt.Sprintf("crc_hyperv_%s.crcbundle", bundleVersion),
	}
}

func GetDefaultBundleForOs(os) string {
	return bundleForOs(version.GetBundleVersion())[os]
}

func GetBundleName(os, bundleVersion string) string {
    return bundleForOs(bundleVersion)[os]
}
```

Reply:
Yes, sure, as long as exported method names is good, we can implement it internally as we prefer :)
Dunno if we should do
```
func GetDefaultBundleForOs(os) string {
	return GetBundleName(os, version.GetBundleVersion())
}
```
for one-line functions, I guess not very important :)